rocky/rocky/templates/organizations/organization_list.html

```diff
@@ -26,16 +26,16 @@ <h1>{% translate 'Organizations' %}</h1>
                 </tr>
               </thead>
               <tbody>
-                {% for object in object_list %}
+                {% for organization in organization_list %}
                   <tr>
                     <td>
-                      <a href="{% url 'organization_detail' organization_code=object.organization.code %}">{{ object.organization.name }}</a>
+                      <a href="{% url 'organization_detail' organization_code=organization.code %}">{{ organization.name }}</a>
                     </td>
                     <td>
-                      <a href="{% url 'organization_detail' organization_code=object.organization.code %}">{{ object.organization.code }}</a>
+                      <a href="{% url 'organization_detail' organization_code=organization.code %}">{{ organization.code }}</a>
                     </td>
-                    <td>{% include 'organizations/organization_tags.html' with organization=object.organization %}</td>
-                    <td>{{ object.total_members }}</td>
+                    <td>{% include 'organizations/organization_tags.html' %}</td>
+                    <td>{{ organization.members.count }}</td>
```

Comment:
This wil do two database queries for each organization, one to get the tags and on to get the member count.

We can do that in two queries: `Organization.objects.annotate(member_count=Count('members')).prefetch_related('tags')` 

`prefetch_related` will do a second query because the tags are a many to many. So the `get_queryset` should in `OrganizationListView` should be something like `Organization.objects.annotate(member_count=Count('members')).prefetch_related('tags').filter(id_in=[organization.id for organization in user.get_organizations())`

Reply:
Done. Including tests :)